### PR TITLE
feat(service-feedback): implement Feedback service

### LINKED ISSUE #10: Create feedback service
**Description**

### PR DESCRIPTION
**Pull request by JonasWanke**
**Fixes: #10**

**Checklist**

- [x] Commit messages follow conventionalcommits.org
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Changes**

### CODE REVIEW

File: service-common/src/main/kotlin/de/hpi/cloud/common/utils/grpc/Builder.kt [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
 
+import com.couchbase.client.java.document.JsonDocument
 import com.couchbase.client.java.document.json.JsonObject
 import com.google.protobuf.GeneratedMessageV3.Builder
+import de.hpi.cloud.common.utils.couchbase.KEY_ID
+import de.hpi.cloud.common.utils.couchbase.KEY_TYPE
 import de.hpi.cloud.common.utils.couchbase.KEY_VALUE
+import de.hpi.cloud.common.utils.couchbase.KEY_VERSION
+
+fun buildJsonDocument(id: String, type: String, version: Int, value: Map<String, Any?>): JsonDocument {
```
Comment (ShirleyNekoDev) [inline service-common/src/main/kotlin/de/hpi/cloud/common/utils/grpc/Builder.kt]: code duplication with Entity

File: service-feedback/build.gradle [hunk 2]
```diff
@@ -0,0 +1,28 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.40")
```
Comment (ShirleyNekoDev) [inline service-feedback/build.gradle]: generalize kotlin version

File: service-feedback/src/main/kotlin/de/hpi/cloud/feedback/FeedbackService.kt [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    "screenUri" to screenUri?.toString(),
+                    "user" to req.feedback.user,
+                    "screenshot" to req.feedback.screenshot.toByteArray().encodeBase64(),
+                    "log" to req.feedback.log
+                )
+            )
+        )
+        Feedback.newBuilder(req.feedback)
+            .setId(id)
+            .build()
```
Comment (ShirleyNekoDev) [inline service-feedback/src/main/kotlin/de/hpi/cloud/feedback/FeedbackService.kt:54]: don't send everything back to the client

**Review by ShirleyNekoDev — CHANGES_REQUESTED** [review]
minor changes - see comments

File: service-feedback/src/main/kotlin/de/hpi/cloud/feedback/FeedbackService.kt (hunk 3 above)
Comment (JonasWanke) [reply service-feedback/src/main/kotlin/de/hpi/cloud/feedback/FeedbackService.kt:54]: Moved to a separate issue: #21

**Pull request merged**